RecoVertex/Configuration/python/RecoVertex_phase2_timing_cff.py

```diff
@@ -31,20 +31,26 @@
                                                           trackTimeResoTag="trackExtenderWithMTD:generalTracksigmat0")
 offlinePrimaryVertices4DnoPIDWithBS=offlinePrimaryVertices4DnoPID.clone(vertices="unsortedOfflinePrimaryVertices4DnoPID:WithBS")
 
-unsortedOfflinePrimaryVertices4DwithPID = unsortedOfflinePrimaryVertices4D.clone(TrackTimesLabel = "tofPID:t0safe",
-                                                                                 TrackTimeResosLabel = "tofPID:sigmat0safe",
+unsortedOfflinePrimaryVertices4DwithPID = unsortedOfflinePrimaryVertices4D.clone(TrackTimesLabel = "tofPID4DnoPID:t0safe",
+                                                                                 TrackTimeResosLabel = "tofPID4DnoPID:sigmat0safe",
                                                                                  )
 trackWithVertexRefSelectorBeforeSorting4DwithPID = trackWithVertexRefSelector.clone(vertexTag="unsortedOfflinePrimaryVertices4DwithPID",
                                                                              ptMax=9e99,
                                                                              ptErrorCut=9e99)
 trackRefsForJetsBeforeSorting4DwithPID = trackRefsForJets.clone(src="trackWithVertexRefSelectorBeforeSorting4DwithPID")
 offlinePrimaryVertices4DwithPID=offlinePrimaryVertices4D.clone(vertices="unsortedOfflinePrimaryVertices4DwithPID",
                                                      particles="trackRefsForJetsBeforeSorting4DwithPID",
-                                                     trackTimeTag="tofPID:t0safe",
-                                                     trackTimeResoTag="tofPID:sigmat0safe")
+                                                     trackTimeTag="tofPID4DnoPID:t0safe",
+                                                     trackTimeResoTag="tofPID4DnoPID:sigmat0safe")
 offlinePrimaryVertices4DwithPIDWithBS=offlinePrimaryVertices4DwithPID.clone(vertices="unsortedOfflinePrimaryVertices4DwithPID:WithBS")
 
 from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import tpClusterProducer
 from SimTracker.TrackAssociatorProducers.quickTrackAssociatorByHits_cfi import quickTrackAssociatorByHits
 from SimTracker.TrackAssociation.trackTimeValueMapProducer_cfi import trackTimeValueMapProducer
-from CommonTools.RecoAlgos.tofPID_cfi import tofPID
+from RecoMTD.TimingIDTools.tofPIDProducer_cfi import tofPIDProducer
+
+tofPID4DnoPID=tofPIDProducer.clone()
+tofPID4DnoPID.vtxsSrc=cms.InputTag('unsortedOfflinePrimaryVertices4DnoPID')
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
```tofPID4DnoPID=tofPIDProducer.clone(vtxsSrc = "unsortedOfflinePrimaryVertices4DnoPID")```